active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,320 @@
+
+This defaults to `Hidden`, but NumberBox offers two visible placement modes: `Inline` and `Compact`. Set `SpinButtonPlacementMode` to `Inline` to enable the buttons to appear beside the control. 
+
+XAML
+```XAML
+<NumberBox Value="{x:Bind Path=ViewModel.NumberBoxValue, Mode=TwoWay}"
+    StepFrequency="2"
+    SpinButtonPlacementMode="Inline" />
+```
+
+![A NumberBox with a down-arrow button and up-arrow button beside it.](images/numberbox-spinbutton-inline.PNG)
+
+Set `SpinButtonPlacementMode` to `Compact` to enable the buttons to appear as a Flyout only when the NumberBox is in focus.  
+
+XAML
+```XAML
+<NumberBox Value="{x:Bind Path=ViewModel.NumberBoxValue, Mode=TwoWay}"
+    StepFrequency="2"
+    SpinButtonPlacementMode="Compact" />
+```
+
+![A NumberBox with a small icon inside of it showing an arrow pointing up and an arrow pointing down.](images/numberbox-spinbutton-compact-non-visible.PNG)
+
+![A NumberBox with a down-arrow button and up-arrow button floating off to the side at an elevated layer.](images/numberbox-spinbutton-compact-visible.PNG)
+
+### Enabling input validation
+
+Setting `ValidationMode` to `InvalidInputOverwritten` will enable NumberBox to overwrite invalid input that is not numerical nor legally formulaic with the last valid value when evaluation is triggered on loss of focus or a press of the "Enter" key.
+
+XAML
+```XAML
+<NumberBox Header="Quantity"
+    Value="{x:Bind Path=ViewModel.NumberBoxValue, Mode=TwoWay}"
+    ValidationMode="InvalidInputOverwritten" />
+```
+
+Setting `ValidationMode` to `Disabled` allows custom input validation to be configured.  
+
+### Formatting input 
+
+[Number formatting](https://docs.microsoft.com/en-us/uwp/api/windows.globalization.numberformatting) can be used to format the value of a Numberbox by configuring an instance of a formatting class and assigning it to the `NumberFormatter` property. Decimal, currency, percent, and significant figures are few of the number formatting classes available. 
+
+Here is an example of using DecimalFormatter to format a NumberBox's value to have one integer digit and two fraction digits:  
+
+XAML
+```XAML
+<NumberBox  x:Name="FormattedNumberBox"
+Value="{x:Bind Path=ViewModel.NumberBoxValue, Mode=TwoWay}" />
+```
+
+C#
+```C#
+private void SetNumberBoxNumberFormatter()
+{
+    DecimalFormatter formatter = new DecimalFormatter();
+    formatter.IntegerDigits = 1;
+    formatter.FractionDigits = 2;
+    FormattedNumberBox.NumberFormatter = formatter;
+}
+```
+
+![A NumberBox showign a value of 0.00.](images/numberbox-formatted.PNG)
+
+### Enabling hyper scroll 
+
+Setting `IsHyperScrollEnabled` to true enables the value of NumberBox to be incremented or decremented by scrolling when a NumberBox has focus and is being hovered over. 
```

Comment:
"HyperScroll" is a new term as far as I can tell.  Searching the tubes turns up some mouse products and a description of this as a "mouse scrollwheel without clickstops."  Should this be `IsMouseScrollEnabled` ?

Reply:
Discussion:
Currently this does not accelerate as you scroll the wheel, so it's not actually hyper yet, though that's the plan.

Recommend: Remove this property, we can add it back later if we find a need for it. Still have the behavior.

Reply:
Resolved.